borg/archiver.py

```diff
@@ -17,6 +18,8 @@
 from . import __version__
 from .archive import Archive, ArchiveChecker, CHUNKER_PARAMS
 from .compress import Compressor, COMPR_BUFFER
+from .logger import create_logger, setup_logging
+logger = create_logger()
```

Comment:
ugh... yeah, i noticed. thing is: things somewhat work now, so i'm hesitant in messing around more with this...

i assume you are refering to:

http://hg.moinmo.in/moin/1.9/file/01e4a21a5d1d/MoinMoin/log.py

?

right now tests pass and output seems generally okay (otherwise identify problems you find).

i would stick with this for now - it will allow us to call logging.foo() instead of print(), which is leagues ahead. then we can improve on this...

again, this is a pretty big diff - i'd like to get it out the door and then improve...